assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -50,8 +50,8 @@ function useUploadVideoFrame({ updateMediaElement }) {
         id: posterId,
         source_url: poster,
         media_details: { width: posterWidth, height: posterHeight },
-      } = await uploadFile(obj);
-      await updateMedia(posterId, {
+      } = await uploadFile(obj, {
+        post: id,
```

Comment:
So we are changing the parent id to the video's id? If so, should we remove this - https://github.com/google/web-stories-wp/pull/1783/files#diff-db3ee3d9878b53a96f21d665e8c79b1bR122

Reply:
Isn't that still relevant for uploading videos & images?

E.g.

Upload image/video -> parent = story
Upload poster image -> parent = video

Reply:
Just checking.